### DISCUSSION TITLE
Need help with LLC for Wemos V4 board

### DISCUSSION THREAD

**Discussion by Alexd911** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#607] [Q&A]
Hey,

I have had some good help and I am getting pointed to the reason my setup is not working is that I need a LLC.

As suggested I have looked through many threads and a tonn of googling but still I can not find a single image showing the 2 boards wired together, eg jumper wires from the LLC to the Wemos board then to the 2 sides of the display board.
I am a total newbie here so I am assuming from disucssion there are 2 boards, a Blue one and a Red one.

So if somone knows the partnumber /description of the board needed and also how to wire it up between the 2 boards then that would be great.

My pump is a 6 wire one.

Cheers

Alex

**Comment by visualapproach** [comment]
double post
